Add missing meta descriptions (#488)
mongodb/docs-golang: source/usage-examples/findOne.txt

Find a Document
===============

.. meta::

>    :description: Retrieve a single document from a collection using the `FindOne()` method in Go, with an example filtering by restaurant name.

Reviewer: ```suggestion
   :description: Retrieve a single document from a collection using the findOne() method in the MongoDB Go Driver.
```
